/**
 * This package contains exception related to the process of validating fields of a User.
 * They are all handled by "UserValidationExceptionHandlers"
 */
package com.xdavide9.sso.exception.user.validation;